sdk/workloads/arm-workloads/test/workloads_examples.ts

```diff
@@ -71,11 +74,70 @@ describe("workloads test", () => {
       routingPreference: "RouteAll",
       tags: { key: "value" }
     };
-    //create monitors
     const res = await client.monitors.beginCreateAndWait(resourceGroup, monitorName, monitorParameter)
     assert.equal(res.name, monitorName);
   });
 
+  //create svi
+  it.only("svi create test", async function () {
```

Comment:
@gargankit-microsoft we met the authorization issue when creating SVI, could you update this test case and relevant recording files directly? So that we could easily reproduce this.

Reply:
Tested locally, working fine. Please go ahead and merge this. I'll test the build again before release